bin/pycbc_single_template

```diff
@@ -317,7 +329,15 @@ with ctx:
         snr, corr, norm = filter.matched_filter_core(template, stilde, 
                                     psd=stilde.psd,
                                     low_frequency_cutoff=flow)
-        snr *= norm
+        snr *= norm 
+                                    
+        if opt.subtract_template:
+            stilde = subtract_template(stilde, template,
+                                snr, opt.trigger_time, opt.low_frequency_cutoff)
```

Comment:
Just double checking. I don't see the recovered phase here, but I assume snr at this point is complex and then you do a complex multiplication above to ensure that the template is subtracted at the correct phase?